modules/gitops-argo-cd-properties.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="argo-cd-properties_{context}"]
+= Argo CD custom resource properties
+
+[role="_abstract"]
+The ArgoCD Custom Resource consists of the following properties:
+
+|===
+|**Name** |**Default** |**Description**
+|`ApplicationInstanceLabelKey` |`mycompany.com/appname`|The `metadata.label` key name where Argo CD injects the app name as a tracking label.
+|`ApplicationSet` |[Object]|`ApplicationSet` controller configuration options.
+|`ConfigManagementPlugins`    |[Empty]|Configuration to add a config management plugin.
+|`Controller`    |[Object]|Argo CD Application Controller options.
+|`Dex`    |[Object]|Dex configuration options.
+|`Disable Admin`    |`false` |Disable the admin user.
+|`GATrackingID`    |[Empty]|The google analytics tracking ID to use.
+|`GAAnonymizeusers`    |[`false` |Enable hashed usernames sent to google analytics.
+|`Grafana`    |[Object]|Grafana configurable options.
+|`HA`    |[Object]|High availablity options.
+|`HelpChatURL`    |`https://mycorp.slack.com/argo-cd` |URL for getting chat help, this will typically be your Slack channel for support.
```

Comment:
```suggestion
|`HelpChatURL`    |`https://mycorp.slack.com/argo-cd` |URL for getting chat help (this will typically be your Slack channel for support).
```